Changelog.md

```diff
@@ -1,11 +1,23 @@
 ### Development
-[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.8.1...master)
+[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.8.2...master)
+
+### 3.8.2 / 2019-01-13
+[Full Changelog](http://github.com/rspec/rspec-rails/compare/v3.8.1...v3.8.2)
 
 Bug Fixes:
 
 * Fix issue with generator for preview specs where `Mailer` would be duplicated
   in the name. (Kohei Sugi, #2037)
 * Fix the request spec generator to handle namespaced files. (Kohei Sugi, #2057)
+* Further truncate system test filenames to handle cases when extra words are
+  prepended. (Takumi Kaji, #2058)
+
+Enhancements
+
+* Use `__dir__` instead of `__FILE__` in generated `rails_helper.rb` where
+  supported. (OKURA Masafumi, #2048)
+* Add `have_enqueued` matcher as a "super" matcher to the `ActiveJob` matchers
+  making it easier to match on `ActiveJob` delivered emails (Joel Lubrano, #2047)
```

Comment:
@benoittgt we don't typically release enhancements in a patch release, so these should be removed

Reply:
I remove them.